packages/styling/src/styles/DefaultSerifs.ts

```diff
@@ -0,0 +1,11 @@
+import { ISerifs } from '../interfaces/index';
+
+export const DefaultSerifs: ISerifs = {
+  depth4: { boxShadow: '0 0 0 0 transparent' },
+  depth8: { boxShadow: '0 3.2px 7.2px 0 rgba(0, 0, 0, 0.132), 0 0.6px 1.8px 0 rgba(0, 0, 0, 0.108)' },
+  depth16: { boxShadow: '0 3.2px 7.2px 0 rgba(0, 0, 0, 0.132), 0 0.6px 1.8px 0 rgba(0, 0, 0, 0.108)' },
+  depth64: { boxShadow: '0 25.6px 57.6px 0 rgba(0, 0, 0, 0.22), 0 4.8px 14.4px 0 rgba(0, 0, 0, 0.18)' },
```

Comment:
Depth values are copied from: packages\fluent-theme\src\fluent\FluentDepths.ts

Reply:
Is it possible to import these from the Fluent theme instead of copying them? While the values are unlikely to change, these could get out of sync in the future.

Reply:
+1

It'd be great to not duplicate values here.

Reply:
I intentionally copied them instead of importing, since the theme should be the "source of truth". Eventually, once we finish the Fluentization work, that project will be going away anyways, right?
(Also, shadows can be added in parallel to the Fluentization work. Unlike rounded corners and colors, it can coexist with old stuff.)